Update functions for add leetcode id and discord id

GuaBaiChi opened the pull request:
functions made:
`table_leetcodeuser_load_by_discord_id`
`table_leetcodeuser_load_by_leetcode_id`
`table_leetcodeuser_loadall`

functions updated:
`table_leetcodeuser_load_by_discord_id`

GuaBaiChi committed:
Update functions for add leetcode id and discord id

jacobmillsfl reviewed (changes requested):
modify code to pass local linter

GuaBaiChi committed:
fixed `bash lint.sh` issues to get 10/10.

Added scripts/commands at the top of `database_util.py for local testing/running

jacobmillsfl commented on src/utils/database_util.py:
```diff
@@ -50,15 +52,41 @@ def table_leetcodequestion_delete(self, question_id: int):
-                table.insert(item)
-                inserts += 1
-        return inserts
+        matches = table.search(where("discord_id") == item["discord_id"] \
+        or where("leetcode_id") == item["leetcode_id"])
```
Please indent the `or where(...)` line by 4 spaces. It is common for multiline code to indent lines after the first one.

GuaBaiChi committed:
indent on line 61

jacobmillsfl commented on src/utils/database_util.py:
```diff
@@ -4,7 +4,9 @@
 from typing import List
 from tinydb import TinyDB, where
-
+# from src.utils.database_util import DatabaseUtil
+# For running/testing locally
+# source bot-env/bin/activate
```
Don't commit test code in comments. This does not belong here

GuaBaiChi committed:
Removed comments related to testing
